pkgs/development/python-modules/pywebview/default.nix

```
@@ -1,14 +1,15 @@
-{ lib, buildPythonPackage, fetchFromGitHub }:
+{ lib, buildPythonPackage, fetchFromGitHub, pythonOlder }:
 
 buildPythonPackage rec {
   pname = "pywebview";
-  version = "3.2";
+  version = "3.3.1";
+  disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.5";
```
It sounds like this package will technically work on python3.5 & python3.6 (it's tested on Travis against 3.5), it just needs an extra package (that's what the error meant). Here are the modifications you'll need to make:

* [ ] Add ``importlib-resources`` argument to the top
* [ ] Add ``propagatedBuildInputs = lib.optionals (pythonOlder "3.7") [importlib-resources];`` at the level of ``src``

---

Implemented these changes. But now i'm back with my problem regarding missing GUI dependencies (see my initial PR which added `pywebview` as a dependency for `jellyfin-mpv-shim`: https://github.com/NixOS/nixpkgs/pull/81817#issuecomment-597247695).

Are you familiar on how to properly add the various dependencies (QT/GTK/...) and maybe give me some hints? 

I tried adding e.g. `pyqt5` to the inputs list but the tests still complained about `ModuleNotFoundError: No module named 'PyQt5'` and so on.

---

I'm not terribly familiar with GTK, but I've dealt a little w/ pyqt5 in the past.

At a guess, the scripts need to be wrapped with the ``wrapQtAppsHook``, but a little bit of trickery is actually needed to run Qt apps with python (i.e. for testing this package).
* Example program (not python): https://github.com/NixOS/nixpkgs/pull/89345/files
* Nix Manual: https://nixos.org/nixpkgs/manual/#sec-language-qt. Note this callout on Python apps: 

> Note: wrapQtAppsHook ignores files that are non-ELF executables. This means that scripts won't be automatically wrapped so you'll need to manually wrap them as previously mentioned. An example of when you'd always need to do this is with Python applications that use PyQT.

Note about GTK: I've had lots of GTK errors thrown (and programs not launch) when trying to run GTK apps on Ubuntu, when the same app works on NixOS. So your OS could be part of the issue.

---

I was trying to look up some example code I had with a python application that did that, but I can't find it right now. Here's it from memory as best as I can...

```nix
{..., wrapQtAppsHook}:

buildPythonApplication{

  preFixup = ''
    for $file in $out/bin/* do
      wrapQtApp $file # NOTE: maybe add something like --prefix PYTHONPATH : $out/${python.sitePackages}?? Can't remember...
    done  
  '';
}
```